test/test_select_multiple.py

```diff
@@ -26,16 +27,16 @@ class TestSelectMultiplePrint(unittest.TestCase):
     def test_list_newlines(self, mock_print):
         args_list = ["foo", "bar"]
         with self.assertRaises(MockException):
-            cutie.select_multiple(args_list)
+            cutie.select_multiple(args_list, hide_confirm=False)
         mock_print.assert_called_once_with("\n" * (len(args_list) - 1))
 
     @mock.patch("cutie.readchar.readkey", side_effect=MockException)
     @mock.patch("cutie.print")
     def test_move_to_first_item(self, mock_print, *m):
         args_list = ["foo", "bar"]
         with self.assertRaises(MockException):
-            cutie.select_multiple(args_list)
-        self.assertEqual(mock_print.call_args_list[1], ((f"\033[{len(args_list) + 2}A",),))
+            cutie.select_multiple(args_list, hide_confirm=False)
```

Comment:
I'd rather modify the tests to test the default behaviour here, so that you don't have to pass `hide_confirm=False` just to make them pass.

Reply:
I changed all of the `select_multiple` tests to test for the default and altered specific ones to now test for `hide_confirm=False`